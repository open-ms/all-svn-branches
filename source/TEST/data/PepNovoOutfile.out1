>> 1 /home/shared/pepnovo/4611_raw_ms2_picked.mzXML.1001.2.dta
#Index	Prob	Score	N-mass	C-Mass	[M+H]	Charge
