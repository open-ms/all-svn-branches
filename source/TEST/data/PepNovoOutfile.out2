>> 0 /home/shared/pepnovo/4611_raw_ms2_picked.mzXML.1000.2.dta
No solutions found.

>> 1 /home/shared/pepnovo/4611_raw_ms2_picked.mzXML.1001.2.dta
#Index	Prob	Score	N-mass	C-Mass	[M+H]	Charge	Sequence
0	0.089	-12.997	362.995	1021.259	1830.158	2	CKPTDN
1	0.085	-12.997	362.995	1021.259	1830.158	2
