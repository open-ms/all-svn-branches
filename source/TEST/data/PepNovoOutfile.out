PepNovo+, Build 20081230
Copyright 2008, The Regents of the University of California. All Rights Reserved.
Created by Ari Frank (arf@cs.ucsd.edu)

Initializing models (this might take a few seconds)... Done.
Fragment tolerance : 0.5000
PM tolernace       : 2.5000
PTMs considered    : K+42:Y+42

*** Warning: searching with unmodified cystine, usually the PTM C+57 should be included ***


>> 0 1
#Index	RnkScr	PnvScr	N-Gap	C-Gap	[M+H]	Charge	Sequence
0	9.888	109.029	0.000	0.000	873.621	2	VKEAMAPK
1	9.236	97.953	99.281	0.000	873.621	2	KEAMAPK
2	8.872	95.284	0.000	0.000	873.621	2	VKEMAAPK
3	8.720	95.559	99.281	0.000	873.621	2	KESDAPK
4	8.640	106.635	0.000	0.000	873.621	2	VKESDAPK
5	8.472	98.366	99.281	0.000	873.621	2	QEAMAPK
6	8.383	94.479	99.281	0.000	873.621	2	KSLMAPK
7	8.348	92.526	0.000	0.000	873.621	2	VKSSDLPK
8	8.321	110.062	0.000	0.000	873.621	2	VKSMALPK
9	8.258	114.657	0.000	0.000	873.621	2	VKSMLAPK
10	8.211	80.165	127.731	0.000	873.031	2	VEAMAPK
11	8.045	92.633	0.000	0.000	873.621	2	GK+42EAMAPK
12	8.032	95.220	0.000	0.000	873.621	2	AGVEAMAPK
13	7.994	87.907	0.000	0.000	873.621	2	VKEATTPK
14	7.951	93.127	0.000	0.000	873.621	2	NLEAMAPK
15	7.918	98.986	99.281	0.000	873.621	2	KSMALPK
16	7.900	94.642	0.000	0.000	873.621	2	VKECVAPK
17	7.871	105.116	0.000	0.000	873.621	2	VKETTAPK
18	7.719	95.971	99.281	0.000	873.621	2	QESDAPK
19	7.632	103.581	99.281	0.000	873.621	2	KSMLAPK

